dashboard/app/views/home/_donor_footer.html.haml

```diff
@@ -0,0 +1,20 @@
+%div{ style: "color: white; text-align: center; padding-top: 30px; background: url(#{asset_path('dashboard_footer-cropped.png')}); background-size: cover; min-height: 340px;" }
```

Comment:
All of this styling should be moved into a `.scss` file, either [application.scss](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/assets/stylesheets/application.scss) or a standalone file alongside it.